cmd/eksctl-anywhere/cmd/applypackages.go

```diff
@@ -43,7 +43,7 @@ var applyPackagesCommand = &cobra.Command{
 
 func applyPackages(ctx context.Context) error {
 	kubeConfig := kubeconfig.FromEnvironment()
-	deps, err := curatedpackages.NewDependenciesForPackages(ctx, kubeConfig)
+	deps, err := NewDependenciesForPackages(ctx, "", "", kubeConfig)
```

Comment:
These should be options if we are passing empty strings

Reply:
@vivek-koppuru, could you give a recommendation for this? I am not aware of any golang construct that allows optional known string.